doc/developer/DeveloperDocumentation.adoc

```diff
@@ -7,14 +7,16 @@ Here is basically what you will need to set up on your development machine:
 * *Create a database and start it:*
   ** (Optional) You can change your db root account's password in the database/instance/manage-db.sh script.
   By default it's `dev_password_to_change`.
-  ** Run `./database/instance/manage-db.sh create <FOLDER_PATH_TO_PERSISTS>`
-  ** Run `mvn liquibase:updateTestingRolling -Pdev -Dliquibase.database.password=<YOUR DB ACCOUNT PASS>`
+  ** Go to the folder containing the script `manage-db.sh`:  `cd ./database/instance`, from the project root
+  ** Run it: `./manage-db.sh create <FOLDER_PATH_TO_PERSISTS>`
+  ** (Optional) You can check that your relational database management system is up and running: `./manage-db.sh mysqladmin`
 * *Build the server part and start it:*
-  ** Run `mvn install -Pdev` in the project's root folder to build the server and its dependencies
-     (you can speed up the build by running `mvn install -Pdev -Dmaven.test.skip=true` instead but you'll
-     lose your Unit tests).
-  ** Run `java -Dspring.profiles.active=dev -jar server/target/ara-server-*.jar -Dspring.datasource.password=<YOUR DB ACCOUNT PASS> -Dspring.config.location=classpath:/` to launch the back-end server. +
-  ** When modifying the server, just run `mvn install -Pdev` in the `server` folder to not rebuild all dependencies, this time.
+  ** Run `mvn install clean -Pdev` in the project's root folder to build the server and its dependencies.
```

Comment:
Revert the command `mvn clean install` and not ìnstall clean` (which is valid but will delete everything it builds 😅 )

Same remark about the `root folder`, use `<ARA_ROOT>` to be precise and consistent :)

Reply:
done